src/main/java/com/minecolonies/util/Utils.java

```diff
@@ -374,6 +394,11 @@ public static int getMiningLevel(@Nullable ItemStack stack, @Nullable String too
         {
             return -1;
         }
+        if (!Compatibility.getMiningLevelCompatibility(stack, tool))
+        {
+            return -1;
+        }
+        //todo: use 'better' version of this thing
         return stack.getItem().getHarvestLevel(stack, tool);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "getHarvestLevel"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)